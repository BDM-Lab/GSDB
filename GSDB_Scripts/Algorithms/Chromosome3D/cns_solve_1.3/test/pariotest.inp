remark   PARIOTEST
remark   =========
remark  
remark   test for parameter I/O
remark   Axel Brunger, 7-JUL-84


parameter
   
   BOND C    C      450.0       1.38  
   BOND C    CH1E   405.0       1.52


   ANGLE C    C    C       70.0     106.5
   ANGLE C    C    CH2E    65.0     126.5
   ANGLE C    CH1E CH1E    70.0     110.0

   DIHEdral X    C    CH1E X        0.0       3       0.0

   IMPRoper C    X    X    C       25.0       0       0.0

   NONBONDED C         0.1200    3.7418      0.1   3.3854
   NONBONDED CH1E      0.0486    4.2140      0.1   3.3854


end


parameter ? end


parameter

   BOND C    CH2E   405.0       1.52   

   ANGLE C    CH1E CH2E    70.0     109.5

   DIHEdral X    C    CH2E X        0.0       3       0.0

   IMPRoper C    X    X    CH2E    90.0       0       0.0

   NONBONDED CH2E      0.1142    3.9823      0.1   3.3854

   NBFIX C    CH1E     0.05       4.0        0.1   4.0


end

parameter ?  end

stop
