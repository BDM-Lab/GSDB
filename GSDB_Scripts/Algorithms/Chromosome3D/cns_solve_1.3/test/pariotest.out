          ============================================================
          |                                                          |
          |            Crystallography & NMR System (CNS)            |
          |                         CNSsolve                         |
          |                                                          |
          ============================================================
           Version: 1.3
           Status: General release
          ============================================================
           Written by: A.T.Brunger, P.D.Adams, G.M.Clore, W.L.DeLano,
                       P.Gros, R.W.Grosse-Kunstleve,J.-S.Jiang,J.M.Krahn,
                       J.Kuszewski, M.Nilges, N.S.Pannu, R.J.Read,
                       L.M.Rice, G.F.Schroeder, T.Simonson, G.L.Warren.
           Copyright (c) 1997-2010 Yale University
          ============================================================
           Running on machine: lewis4-r710-login-node223 (x86_64/Linux,64-bit)
                               with 16 threads
           Program started by: oeow39
           Program started at: 16:08:05 on 05-Jul-2018
          ============================================================

 FFT3C: Using FFTPACK4.1

 CNSsolve>remark   PARIOTEST 
 CNSsolve>remark   ========= 
 CNSsolve>remark 
 CNSsolve>remark   test for parameter I/O 
 CNSsolve>remark   Axel Brunger, 7-JUL-84 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR> 
 PARRDR>   BOND C    C      450.0       1.38 
 PARRDR>   BOND C    CH1E   405.0       1.52 
 PARRDR> 
 PARRDR> 
 PARRDR>   ANGLE C    C    C       70.0     106.5 
 PARRDR>   ANGLE C    C    CH2E    65.0     126.5 
 PARRDR>   ANGLE C    CH1E CH1E    70.0     110.0 
 PARRDR> 
 PARRDR>   DIHEdral X    C    CH1E X        0.0       3       0.0 
 PARRDR> 
 PARRDR>   IMPRoper C    X    X    C       25.0       0       0.0 
 PARRDR> 
 PARRDR>   NONBONDED C         0.1200    3.7418      0.1   3.3854 
 PARRDR>   NONBONDED CH1E      0.0486    4.2140      0.1   3.3854 
 PARRDR> 
 PARRDR> 
 PARRDR>end 
 CNSsolve> 
 CNSsolve> 
 CNSsolve>parameter ? end 

 BOND  C    C        450.000 {sd=     0.026}      1.380
 BOND  C    CH1E     405.000 {sd=     0.027}      1.520

 ANGLe  C    C    C         70.00 {sd=     3.726}    106.5000
 ANGLe  C    C    CH2E      65.00 {sd=     3.867}    126.5000
 ANGLe  C    CH1E CH1E      70.00 {sd=     3.726}    110.0000

 DIHEdral  X    C    CH1E X           0.00 {sd=999999.000}    3      0.0000

 IMPRoper  C    X    X    C          25.00 {sd=     6.235}    0      0.0000

 !                  eps     sigma       eps(1:4) sigma(1:4)
 NONBonded  C       0.1200   3.7418      0.1000   3.3854
 NONBonded  CH1E    0.0486   4.2140      0.1000   3.3854

 CNSsolve> 
 CNSsolve> 
 CNSsolve>parameter 
 PARRDR> 
 PARRDR>   BOND C    CH2E   405.0       1.52 
 PARRDR> 
 PARRDR>   ANGLE C    CH1E CH2E    70.0     109.5 
 PARRDR> 
 PARRDR>   DIHEdral X    C    CH2E X        0.0       3       0.0 
 PARRDR> 
 PARRDR>   IMPRoper C    X    X    CH2E    90.0       0       0.0 
 PARRDR> 
 PARRDR>   NONBONDED CH2E      0.1142    3.9823      0.1   3.3854 
 PARRDR> 
 PARRDR>   NBFIX C    CH1E     0.05       4.0        0.1   4.0 
 PARRDR> 
 PARRDR> 
 PARRDR>end 
 CNSsolve> 
 CNSsolve>parameter ?  end 

 BOND  C    C        450.000 {sd=     0.026}      1.380
 BOND  C    CH1E     405.000 {sd=     0.027}      1.520
 BOND  C    CH2E     405.000 {sd=     0.027}      1.520

 ANGLe  C    C    C         70.00 {sd=     3.726}    106.5000
 ANGLe  C    C    CH2E      65.00 {sd=     3.867}    126.5000
 ANGLe  C    CH1E CH1E      70.00 {sd=     3.726}    110.0000
 ANGLe  C    CH1E CH2E      70.00 {sd=     3.726}    109.5000

 DIHEdral  X    C    CH1E X           0.00 {sd=999999.000}    3      0.0000
 DIHEdral  X    C    CH2E X           0.00 {sd=999999.000}    3      0.0000

 IMPRoper  C    X    X    C          25.00 {sd=     6.235}    0      0.0000
 IMPRoper  C    X    X    CH2E       90.00 {sd=     3.286}    0      0.0000

 !                  eps     sigma       eps(1:4) sigma(1:4)
 NONBonded  C       0.1200   3.7418      0.1000   3.3854
 NONBonded  CH1E    0.0486   4.2140      0.1000   3.3854
 NONBonded  CH2E    0.1142   3.9823      0.1000   3.3854

 NBFIx  C    CH1E        0.050        4.000           0.100        4.000
 CNSsolve> 
 CNSsolve>stop 
          ============================================================
           Maximum dynamic memory allocation:     8248136 bytes
           Maximum dynamic memory overhead:           288 bytes
           Program started at: 16:08:05 on 05-Jul-2018
           Program stopped at: 16:08:05 on 05-Jul-2018
           CPU time used:       0.0044 seconds
          ============================================================
